src/promptflow/tests/sdk_cli_azure_test/conftest.py

```diff
@@ -34,28 +35,32 @@ def ml_client(
 
 @pytest.fixture()
 def remote_client() -> PFClient:
-    return PFClient(
-        credential=get_cred(),
-        subscription_id="96aede12-2f73-41cb-b983-6d11a904839b",
-        resource_group_name="promptflow",
-        workspace_name="promptflow-eastus",
-    )
+    # enable telemetry for CI
+    with environment_variable_overwrite("TELEMETRY_ENABLED", "true"):
```

Comment:
`TELEMETRY_ENABLED` set as a common constant? Seems it's used in multiple places